components/header.php

```diff
@@ -1,8 +1,7 @@
 <header class="header">
         <nav class="navbar">
             <ul class="navbar-ul base">
-                <li><a class="link" href="/">Home</a></li>
-                <li><a class="link" href="/about.php">About</a></li>
+            <?php require 'components/links.php'; ?>
```

Comment:
By making the links in a separate file you can include it many times, as you are doing in the footer. It is great that you see opportunities to reuse code and practice DRY proactively, great job!